mixer/adapter/stackdriver/metric/bufferedClient.go

```diff
@@ -41,9 +52,12 @@ type buffered struct {
 	closeMe io.Closer
 
 	// Guards buffer
-	m                   sync.Mutex
-	buffer              []*monitoringpb.TimeSeries
+	m      sync.Mutex
+	buffer []*monitoringpb.TimeSeries
+	// Retry buffer is keyed by hash(metric name, monitored resource).
+	retryBuffer         map[uint64]retryTimeSeries
```

Comment:
CreateTimeSeries API has a constrain that only it could only contain one data point, so all timeSeries were merged based on metric name and monitored resource before sending into one point, so it is not necessary to keep a slice.

Reply:
I'm suggesting a slice instead of a map, not in addition to a map.

Reply:
I see, you are saying no need to merge failed timeseries with the newer one. Just send them together with order, right?

Reply:
I think it's acceptable if a point gets dropped in that circumstance, and it should make the code much simpler to use a slice here.

Reply:
Done.